README.md

```diff
@@ -76,8 +76,11 @@ git submodule init && git submodule update
 To compile, starting at the project root directory, create the build directory and Makefile:
 
 ```
-mkdir build && cd build && cmake ..
+mkdir build && cd build
+cmake ..
 ```
+If you are on mac os, change the cmake command to be `cmake .. -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl`
```

Comment:
What about `-DOPENSSL_ROOT_DIR=$(brew --prefix openssl)`?

`mac os` -> `macOS`

Additionally, you switched the `WITH_PROCPS` option to `OFF` by default. What about either:
- Add an explicit flag to your command for Mac users; i.e. `cmake .. -DOPENSSL_ROOT_DIR=$(brew --prefix openssl) -DWITH_PROCPS=OFF`, or
- Keep the option disabled by default, but add a comment above this option in the `CMakeLists.txt` file in order to explicitly mention that this option does not work on macOS (since there is no `/proc`). In fact; that would certainly avoid confusion if someone tries to switch the "memory profiling option" to `ON`
?

Reply:
> Additionally, you switched the WITH_PROCPS option to OFF by default. What about either [...]

Maybe we can even add a block to check if the platform macOS if `WITH_PROCPS` is enabled and display a warning message.
Something like:
```bash
if("${WITH_PROCPS}")
  if(APPLE)
    message(WARNING "WITH_PROCPS must be set to OFF for macOS build")
  endif()
  include(FindPkgConfig)
  pkg_check_modules(
    PROCPS
    REQUIRED

  libprocps
  )
else()
  add_definitions(
    -DNO_PROCPS
)
endif()
```

Reply:
Great suggestions, thank you! I'll apply all of these!

I feel like its better to keep with_procps off by default, since I imagine needing procps is relatively uncommon amongst people using/working on the library.